static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -187,7 +187,7 @@ const makeABTestVariant = (
             parentTest.campaignSuffix
         ),
         contributeURL = addTrackingCodesToUrl({
-            base: contributionsBaseURL,
+            base: options.contributionsBaseURL || contributionsBaseURL,
```

Comment:
While this isn't needed for this test, it seemed to make sense to also be able to do it for the contribute url too. Thoughts, anyone?

Reply:
Personally I think I might leave it out under the principle of YAGNI

Reply:
You're probably right. I imagined myself in a position of doing something with this in the future and wondering why there's yet another inconsistency in the way things are done.